### DISCUSSION TITLE
Alerts for failed tasks or high volume of jobs.

### DISCUSSION THREAD

**Discussion by kashyap2108** [riverqueue/river#582] [Q&A]
Do we mechanism of sending alerts to slack or email for anomaly scenarios ?

**Comment by bgentry** [comment]
Hi @kashyap2108, check out the [subscriptions](https://riverqueue.com/docs/subscriptions) feature. You can subscribe to events for failed jobs and specifically look at those with a `JobStateDiscarded` to take action on these jobs.

You’ll want that logic to be configured on all your River clients that are working jobs, because subscriptions are only for events local to a given client.
